python/core/qgsrange.sip

```diff
@@ -147,15 +147,19 @@ class QgsTemporalRange
 %End
   public:
 
-    QgsTemporalRange( const T &begin, const T &end, bool includeBeginning = true, bool includeEnd = true );
+#if SIP_VERSION >= 0x41303
 %Docstring
  Constructor for QgsTemporalRange. The ``begin`` and ``end`` are specified,
  and optionally whether or not these bounds are included in the range.
```

Comment:
nope, it's not exactly what was meant here (by @nyalldawson). It's includeBegining/End which specify if bounds are included in the range. begin and end are optional in cpp, mandatory in python (SIP<4.19.3).